rtb/norme_di_progetto_ver0.6.1.typ

```diff
@@ -531,9 +564,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+
+== Miglioramento
+=== Descrizione
+Secondo lo _standard_ ISO/IEC 12207:1995, il processo di miglioramento nel ciclo di vita del _software_ è finalizzato a stabilire, misurare, controllare e migliorare i processi che lo compongono. L’attività di miglioramento è composta da:
+- Analisi: identificare le aree di miglioramento dei processi.
+- Miglioramento: implementare le modifiche necessarie per migliorare i processi di sviluppo del _software_.
+
+==== Analisi dei processi
+Innanzitutto occorre stabilire una serie di processi organizzativi per l'intero ciclo di vita del _software_ applicabili alle varie attività di progetto. Quest'ultimi devono essere documentati, come viene fatto proprio qui, e va implementato un meccanismo di controllo per sviluppare, monitorare e migliorare i processi stessi. Per garantire efficacia e continuo miglioramento dei processi alla fine di ogni _sprint_ il gruppo effettuerà una retrospettiva del lavoro svolto nel periodo trascorso, che verrà verbalizzata dal responsabile nel documento Piano di Progetto.
+
+==== Miglioramento dei processi
+Una volta identificati i potenziali miglioramenti questi vanno effettivamente implementati secondo le corrette a adeguate metodologie. Nella retrospettiva seguente all'applicazione di queste migliorie verrà effettuato un altro _#rifGlossario("feedback")_ a riguardo per progredire ciclicamente.
+
+=== Metriche
+// TODO: Bisogna capire bene quali sono nostre metriche, aspettando il PdQ
+
+== Formazione
+=== Descrizione
+Alla base della buona riuscita del progetto ci sono persone adeguatamente formate a mano a mano che il progetto evolve. Per questo diventa fondamentale il processo di formazione.
+Con esso vengono definiti gli standard e le metodologie con i quali ognuno sarà sempre pronto nell'argomento adeguato nel momento opportuno.
+
+=== Descrizione
+Per iniziare a formare i membri del gruppo, è necessario prima comprendere completamente il dominio del problema. Quindi, è necessario comprendere quali argomenti sono necessari per essere approfonditi e quali abilità sono necessarie per i vari processi. \
+Si deve quindi passare all'individuazione del materiale di formazione, che aumenterà nel tempo poiché il nostro livello di conoscenza e comprensione del problema dovrà aumentare man mano che il progetto avanza.\
+Infine, tutti i membri del gruppo devono imparare da dove e cosa studiare, e devono anche aggiornarsi individualmente, se possibile, con l'aiuto di altri membri più esperti.
+
+=== Strumenti
+Al fine di agevolare il processo di formazione, ogni membro del gruppo incaricato in un determinato _sprint_ di approfondire un determinato argomento dovrà essere in grado di tramandare la propria conoscenza agli altri. 
+Per aiutarci nel processo di apprendimento l'azienda Sync Lab ci fornisce, duranete lo svolgimento del progetto, meteriali quali video e risorse utili alle tecnologie di dominio d'uso del nostro capitolato, nonché delle sedute di _deep dive_ concordate a calendario.
```

Comment:
a parte le prime due virgole da rimuovere (è diventato un while true ormai), typo in "duranete" e "meteriali"